uaclient/security_status.py

```diff
@@ -266,6 +266,7 @@ def get_reboot_status():
         and our_kernel_version == lp_status.kernel
         and lp_status.livepatch is not None
         and lp_status.livepatch.state == "applied"
+        and lp_status.supported == "supported"
```

Comment:
@kian99 Can you confirm that if these two conditions are true, then we can give our "in-between" reboot status (indicating livepatch has the kernel covered for high/critical CVEs)?

Reply:
Yes, according to the original UA [spec](https://docs.google.com/document/d/1Y9qs8JQAOFleB9qebAgD_J1YJ_UDl67zlvTFbiLuTYw/edit#) you only want to give that "in-between" status when patches have been applied and the kernel is still within support so this is the right way to check for that.

I didn't perhaps have the insight to mention this previously but a value of `lp_status.livepatch.state == "nothing-to-apply"` alongside `lp_status.supported == "supported"` would indicate that livepatches will come through for high/critical CVEs even though none have come through yet. Just mentioning this as it might be something you want down the road (a fair portion of the time, machines won't actually have Livepatches applied because there are no high/critical CVEs but the system will still receive patches).

Reply:
Thank you @kian99!

That's an excellent point about the `nothing-to-apply` state. That seems to be the same situation and I maybe we should allow that state to trigger the "in-between" reboot status as well. @lucasmoura Thoughts? (might be worth a quick email and/or sync to make sure all stake holders are okay with that)